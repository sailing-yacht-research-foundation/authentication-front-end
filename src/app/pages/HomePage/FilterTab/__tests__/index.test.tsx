import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { FilterTab } from '../index';

const shallowRenderer = createRenderer();

describe('<FilterTab />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<FilterTab />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
