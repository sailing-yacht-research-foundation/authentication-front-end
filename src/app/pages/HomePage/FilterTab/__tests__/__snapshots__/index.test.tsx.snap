// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilterTab /> should render and match the snapshot 1`] = `
<styled.div>
  <FilterResult />
</styled.div>
`;
